addons/web/static/src/core/position_hook.js

```diff
@@ -101,6 +103,12 @@ function getBestPosition(reference, popper, { container, margin, position }) {
     const refBox = reference.getBoundingClientRect();
     const contBox = container.getBoundingClientRect();
 
+    // Get the offsets of the containing block (relative to the viewport)
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block
+    const popStyle = window.getComputedStyle(popper);
+    const verticalOffset = (popBox.top - parseFloat(popStyle.top)) || 0;
+    const horizontalOffset = (popBox.left - parseFloat(popStyle.left)) || 0;
```

Comment:
If I am not wrong, as we already know the popper style at this point, we could simply do nothing here:
```suggestion
```

And below, in getPositioningData:
```js
return {
    top: positioning.top - popBox.top,
    left: positioning.left - popBox.left,
    direction: DIRECTIONS[d],
    variant: VARIANTS[v],
};
```